app/views/wiki/index.html.erb

```diff
@@ -13,12 +13,12 @@
   <% elsif params[:action] == "liked" %>
 
     <h2>Highly liked wiki pages <small>collaborative documentation by contributors like you</small></h2>
-    <p>&nbsp;<a style="margin-bottom:4px;" href="/wiki/new" class="btn btn-default"><i class="fa fa-plus-circle"></i> Create a new wiki page</a> <span style="margin-top:4px;" class="pull-right"><a href="/wiki/">by recency</a> | <a href="/wiki/popular">by popularity</a> | <span style="color:#aaa">by # of likes</span></p>
+    <p>&nbsp;<a style="margin-bottom:4px;" href="/wiki/new" class="btn btn-default"><i class="fa fa-plus-circle"></i> Create a new wiki page</a> <span style="margin-top:4px;" class="pull-right"><a href="/wiki/">by recency</a> | <a href="/wiki/popular">by popularity</a> | <span style="color:#aaa">by # of likes</span></p></span>
```

Comment:
Here, the `span` tag has to go inside the `p` tag -- the tests complain :-)

Thanks, good catch on missing close span!